filters/remove_frequent_patterns.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+"""
+Removes patterns that occur usually only in the source or in the target. There are two way to set up a pattern:
+ - a two columns type - pattern tab replacement, which is applied to both source and target
+ - a three columns tpye - pattern_on_both_cols tab pattern tab replacement
+    The `pattern_on_both_cols` is matched against tab separated source and target.
+    If it matches then the `pattern` is replaced in both columns.
+
+You can check the three column filter from the examples by running
+```
+mtdata get -l ces-eng -tr ELRC-czech_supreme_audit_office_2018_reports-1-ces-eng  --compress -o data
+```
+and using the `remove_frequent_patterns` filter.
+Then e.g. the line containing `Increasing support for informal carers,` will not start with `• `,
+because the source doesn't start with a bullet either.
+"""
+
+from dataclasses import dataclass
+import re
+import sys
+import argparse
+from typing import List, Optional
+
+
+@dataclass
+class Pattern:
+    group_match: re.Pattern
+    replacement: str
+    pattern_on_both_cols: Optional[re.Pattern] = None
+
+
+def load_patterns(file_path: str) -> List[Pattern]:
+    with open(file_path) as f:
```

Comment:
I didn't catch the `FileNotFoundError` since I would re-raise it anyway. What is the best approach to such errors in filters?

Reply:
This is fine! Filters show their stderr and will interrupt the pipeline when they exit with a non-zero exit status. So just letting the exception pass through all the way to the runtime is a simple way of communicating that.